### ISSUE TITLE
middle-click on icon should toggle "allow all"

### ISSUE THREAD

**Issue by azrdev** [RequestPolicyContinued/requestpolicy#793]
# Steps to reproduce:

1. Click with the middle mouse button on the RP toolbar icon

# What happens?

2. nothing

# What should happen?

3. anything. I propose to enable/disable the addon, i.e. toggle the "allow all requests" state. This makes even more sense given that this state is also indicated by that same icon.

**Comment by myrdd** [comment]
I don't agree having the default action on middle-click being "allow all requests". Still, we could make the action configurable – I'd like that feature.
